### PR TITLE
Node properties

### PR DESCRIPTION
**Pull request by Gayatriuparate**

### CODE REVIEW

File: src/browser/modules/EditorInfo/EditorInfo.jsx [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+        <EditNodes
+          nodeProperties={this.props.nodeProperties._fields[0].properties}
+        />
+      </div>
+    )
+  }
+}
+const mapStateToProps = state => {
+  return {
+    nodeProperties: state.itemEditor.selectedItem
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditorInfo.jsx]: rename nodeProperties to selectedItem

File: src/browser/modules/EditorInfo/EditorInfo.jsx [hunk 2]
```diff
@@ -1,15 +1,27 @@
 /*
- * This program depicts the behaviour of the edit drawer.
+ * This model depicts the behaviour of the edit drawer.
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditorInfo.jsx]: replace model to module

File: src/browser/modules/EditorInfo/EditNodes.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+/*
+ * This model depicts the behaviour of the edit drawer that displays the node relationship
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditNodes.jsx]: replace model to module

File: src/browser/modules/EditorInfo/utils.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { v1 as neo4j } from 'neo4j-driver'
+export const getStringValue = value => {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js:8]: write JSdoc for getStringValue method

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/EditNodes.jsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+import * as _ from 'lodash'
+import { getStringValue } from './utils'
+
+export class EditNodes extends Component {
+  render () {
+    let content = null
+    content = _.map(this.props.nodeProperties, (value, key) => {
+      return (
+        <div>
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditNodes.jsx]: add key attribute to this div 
key={key}

refer
https://reactjs.org/docs/lists-and-keys.html#keys

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/EditorInfo.jsx (hunk 1 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditorInfo.jsx]: renamed to selectedItem

File: src/browser/modules/EditorInfo/EditorInfo.jsx (hunk 2 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditorInfo.jsx]: replaced model to module

File: src/browser/modules/EditorInfo/EditNodes.jsx (hunk 3 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: replaced model to module

File: src/browser/modules/EditorInfo/utils.js (hunk 4 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/utils.js:8]: Added JSdoc for getStringValue method

File: src/browser/modules/EditorInfo/EditNodes.jsx (hunk 5 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: Added key attribute

File: src/browser/modules/EditorInfo/utils.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+/**
+ * getStringValue method checks the datatype of Node properties value and return
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js]: Correct the english

File: src/browser/modules/EditorInfo/EditorInfo.jsx [hunk 7]
```diff
@@ -1,15 +1,27 @@
...
 import { withBus } from 'react-suber'
+import { EditNodes } from './EditNodes'
 
 export class EditorInfo extends Component {
   render () {
-    return <div>Editor</div>
+    return (
+      <div>
+        <EditNodes
+          selectedItem={this.props.selectedItem._fields[0].properties}
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditorInfo.jsx]: selectedItem would be nodeProperties

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/utils.js (hunk 6 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/utils.js]: Corrected

File: src/browser/modules/EditorInfo/EditorInfo.jsx (hunk 7 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditorInfo.jsx]: Changed

File: src/browser/modules/EditorInfo/utils.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
+/**
+ * returns the string representation
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js]: Given the neo4j property data type, transform the property value to string form.

File: src/browser/modules/EditorInfo/utils.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+ * of the requested node property value
+ * @param value
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js:3]: Neo4j property value with valid property data type

File: src/browser/modules/EditorInfo/utils.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+ * value has neo4j data type
+ */
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js:6]: @return {string} String representation of neo4j property value if valid data type, else empty string

File: src/browser/modules/EditorInfo/EditNodes.jsx [hunk 11]
```diff
@@ -0,0 +1,46 @@
+/*
+ * This module depicts the behaviour of the edit drawer that displays the node relationship
+ properties
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditNodes.jsx]: Every line of JSDoc in multiline form should start with '*'

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/utils.js (hunk 8 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/utils.js]: done

File: src/browser/modules/EditorInfo/utils.js (hunk 9 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/utils.js:3]: Replaced

File: src/browser/modules/EditorInfo/utils.js (hunk 10 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/utils.js:6]: Added @return

File: src/browser/modules/EditorInfo/EditNodes.jsx (hunk 11 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: done

Comment (rahul-tarafdar-synerzip) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: > done

not done

File: src/browser/modules/EditorInfo/EditNodes.jsx [hunk 12]
```diff
@@ -0,0 +1,46 @@
+/*
+ * This module depicts the behaviour of the edit drawer that displays the node relationship
+ *properties
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditNodes.jsx]: space between '*' and 'properties'

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/EditNodes.jsx (hunk 12 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: added

File: src/browser/modules/EditorInfo/EditNodes.jsx (hunk 11 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: Missing changes. Added

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**